Update one-click_deploy.rst - Enabled USB debugging on android devices. (#5044)
godotengine/godot-docs: tutorials/export/one-click_deploy.rst

Using one-click deploy
----------------------

> - If deploying to Android, enable developer mode on your mobile device then enable USB debugging.

Reviewer: ```suggestion
- If deploying to Android, enable developer mode on your mobile device then enable USB debugging in the device's settings.
```

> - If deploying to Android, enable developer mode on the your mobile device and turn on USB debugging. 
> - Then connect the device to your PC using an USB cable.

Reviewer: ```suggestion
- If deploying to Android, enable developer mode on your mobile device then enable USB debugging.
- After enabling USB debugging, connect the device to your PC using an USB cable.
```
